tests/integration/components/offering-form-test.js

```diff
@@ -36,13 +36,6 @@ module('Integration | Component | offering form', function (hooks) {
     assert.ok(component.location.hasError);
   });
 
-  test('room validation succeeds on blank value', async function (assert) {
```

Comment:
Can't tell why this test got removed, superfluous now?

Reply:
removal by accident. i added it back in. :heavy_check_mark: